cumulusci/core/github.py

```diff
@@ -158,10 +153,15 @@ def get_auth_from_service(host, keychain) -> tuple:
 def validate_service(options: dict) -> dict:
     username = options["username"]
     token = options["token"]
-    # For backwards compatability, set a default repo_domain
-    repo_domain = options.get("repo_domain", "github.com")
+    # Github service doesn't have "repo_domain",
+    repo_domain = options.get("repo_domain", None)
+
     gh = _determine_github_client(repo_domain, {"token": token})
 
+    default = options.get("default", False)
+    if repo_domain is not None and default:
+        raise GithubException("GitHub Enterprise services may not be set as default.")
```

Comment:
❓: Well, wait a second... now I'm confused. I thought we wanted to move this into its own service so that we _could_ allow setting a default for the Enterprise Server service?

Reply:
W-11413242 created to enhance default behavior